phys2bids/viz.py

```diff
@@ -14,9 +14,12 @@
 def plot_trigger(time, trigger, fileprefix, tr, thr, num_timepoints_expected,
                  filename, figsize=FIGSIZE, dpi=SET_DPI):
     """
-    Produces a textfile of the specified extension `ext`,
-    containing the given content `text`.
-
+    Produces a figure with three plots:
+    1. Plots the trigger in blue, a block in orange that indicates the start and end of the time,
+    red line for the threshold used for trigger detection
+    2. Same plot but this showing only the intial trigger
+    3. Same plot but this showing only the intial trigger
```

Comment:
Points 2 and 3:
2. Same plot but showing only the initial trigger
3. Same plot but showing only the ending trigger 

(^Assume that is what you meant?)